src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -303,11 +303,10 @@ export const useSwapper = () => {
           sellAmount: amount,
           sellAsset,
           buyAsset,
-          sellAssetAccountNumber: 0, // TODO: remove hard coded accountId when multiple accounts are implemented
-          buyAssetAccountNumber: 0, // TODO: remove hard coded accountId when multiple accounts are implemented
           wallet,
           sendMax: false,
           receiveAddress,
+          bip44Params: chainAdapter.buildBIP44Params({ accountNumber: 0 }),
```

Comment:
i think `accountNumber` always defaults to 0 and you can just pass an empty object

```suggestion
          bip44Params: chainAdapter.buildBIP44Params({}),
```

Reply:
outdated